test/index.js

```diff
@@ -243,7 +248,12 @@ describe('hapi-setup Plugin', function () {
           },
           attributes: {
             name: 'bar',
-            version: '1.0.0'
+            version: '1.0.0',
+            pkg: {
+              version: '0.0.0'
```

Comment:
I see what you mean, the data is the same in both cases, however expect seems to think it's different.

```
Failed tests:

  4) hapi-setup Plugin returns plugin information:

      actual expected

      {
        "name": "foo",
        "version": "0.0.0",
        "multiple": false,
        "options": {},
        "attributes": {
          "name": "foo",
          "pkg":
        {
            "version": "0.0.0"
          },
          "multiple": false,
          "connections": true
        }
      }

      Expected { name: 'foo',
  version: '0.0.0',
  multiple: false,
  options: {},
  attributes: 
   { name: 'foo',
     pkg: { version: '0.0.0' },
     multiple: false,
     connections: true } } to equal specified value

      at /Users/abrons/devel/adac/hapi-setup/test/index.js:228:37
```